monkey/infection_monkey/monkey.py

```diff
@@ -72,26 +72,31 @@ def initialize(self):
         arg_parser = argparse.ArgumentParser()
         arg_parser.add_argument('-p', '--parent')
         arg_parser.add_argument('-t', '--tunnel')
-        arg_parser.add_argument('-s', '--server')
+        arg_parser.add_argument('-s', '--servers', nargs = '+')
         arg_parser.add_argument('-d', '--depth', type=int)
         self._opts, self._args = arg_parser.parse_known_args(self._args)
 
         self._parent = self._opts.parent
         self._default_tunnel = self._opts.tunnel
-        self._default_server = self._opts.server
+        self._default_servers = self._opts.server
 
         if self._opts.depth:
             WormConfiguration._depth_from_commandline = True
         self._keep_running = True
         self._network = NetworkScanner()
         self._dropper_path = sys.argv[0]
 
-        if self._default_server:
-            if self._default_server not in WormConfiguration.command_servers:
-                LOG.debug("Added default server: %s" % self._default_server)
-                WormConfiguration.command_servers.insert(0, self._default_server)
-            else:
-                LOG.debug("Default server: %s is already in command servers list" % self._default_server)
+        server_num = 0   
+        server_count = len(self._default_servers)
+  
+        if self._default_servers:
+         while server_num < (server_count):
+           if self._default_servers[server_num ] not in WormConfiguration.command_servers:
```

Comment:
Isn't `WormConfiguration.command_servers` empty/contains default values at this point? Why don't you use list comprehension then?